src/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -226,4 +224,228 @@
+  </labels>
+  <labels>
+    <fullName>cookbook_ProvideDetailsTitle</fullName>
+    <value>Speed up the resolution of your problem</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>Speed up the resolution of your problem</shortDescription>
+  </labels>
+  <labels>
+    <fullName>cookbook_ProvideDetailsSubtitle</fullName>
+    <value>Detail your problem to find the agent best suited to help you</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>Detail your problem to find the agent best suited to help you</shortDescription>
+  </labels>
+  <labels>
+    <fullName>cookbook_PriorityLabel</fullName>
+    <value>How would you classify the priority of your issue?</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>How would you classify the priority of your issue?</shortDescription>
+  </labels>
+  <labels>
+    <fullName>cookbook_TypeLabel</fullName>
+    <value>What type of problem is it?</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>What type of problem is it?</shortDescription>
+  </labels>
+  <labels>
+    <fullName>cookbook_ReasonLabel</fullName>
+    <value>What's the reason behind opening this case?</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>What's the reason behind opening this case?</shortDescription>
+  </labels>
+  <labels>
+    <fullName>cookbook_MoreOptions</fullName>
+    <value>More options</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>More options</shortDescription>
+  </labels>
+  <labels>
+    <fullName>cookbook_TheseResourcesMightHelp</fullName>
+    <value>These resources might help</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>These resources might help</shortDescription>
+  </labels>
+  <labels>
+    <fullName>cookbook_ResourcesSuggested</fullName>
+    <value>Resources suggested based on your problem description</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>Resources suggested based on your problem description</shortDescription>
+  </labels>
+  <labels>
+    <fullName>cookbook_SolvedProblem</fullName>
+    <value>Solved your problem?</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>Solved your problem?</shortDescription>
+  </labels>
+  <labels>
+    <fullName>cookbook_StillNeedHelp</fullName>
+    <value>Still need help?</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>Still need help?</shortDescription>
+  </labels>
+  <labels>
+    <fullName>cookbook_ChangedYourMind</fullName>
+    <value>Changed your mind?</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>Changed your mind?</shortDescription>
```

Comment:
Didn't find the label for this one but under "Change your mind?", please update button as follows:
"Abandon Request"-->"Abandon request"